chore: Style and SEO updates (#3775)
hashicorp/terraform-cdk: website/docs/cdktf/create-and-deploy/environment-variables.mdx

---

> page_title: Environment variables

Reviewer: ```suggestion
page_title: CDK for Terraform environment variables
```

> page_title: CDK for Terraform environment variables

Author: ```suggestion
page_title: CDKTF environment variables
```

description: >-

>   Use environment variables to configure the behavior of the CDK for Terraform

Author: ```suggestion
  Use environment variables to configure the behavior of the CDKTF
```

  CLI.
---

> # CDK for Terraform environment variables

Author: ```suggestion
# CDKTF environment variables
```
